tests/widgets/test_results_table.py

```diff
@@ -0,0 +1,122 @@
+import pytest
```

Comment:
I'd rename this to `test_results.py`, although I'd prefer a better name for the source module...

Reply:
Renamed to something completely different!